### PR TITLE
Sara/Modifying help centre

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   Modifying and adding some content

## Type of change

-   [x] Bug fix
-   [x] New feature
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update

### CODE REVIEW

File: src/pages/help-centre/_help-articles.js [hunk 1]
```diff
@@ -222,6 +434,40 @@ export const articles = [
...
+        category: localize('Privacy and security'),
+        articles: [
+            {
+                title: localize(
+                    'How we safeguard your account when you change or reset your Deriv password',
+                ),
+                category: localize('Privacy and security'),
+                sub_category: localize('Safeguard'),
+                label:
+                    'How-we-safeguard-your-account-when-you-change-or-reset-your-Deriv-password',
```
Comment (mustofa-binary) [inline src/pages/help-centre/_help-articles.js]: can simplify this to safeguard-reset-password, and so do others

File: src/pages/help-centre/_help-articles.js [hunk 2]
```diff
@@ -109,6 +114,111 @@ const AuthenticatingYourAccount = () => (
...
+        <Text>
+            {localize(
+                'To change your Deriv password if you’re already logged in:',
+            )}
+        </Text>
+        <Ol>
+            <Li>
+                <Localize
+                    translate_text="Go to <0>Settings > Security and safety > Deriv password.</0>"
+                    components={[<BoldText key={0} />]}
```
Comment (mustofa-binary) [inline src/pages/help-centre/_help-articles.js]: you can use `<strong>` as well

File: src/pages/help-centre/_help-articles.js (hunk 1 above)
Comment (sara-fs) [reply src/pages/help-centre/_help-articles.js]: Thank you Mustofa, if it is acceptable to make it shorter, so I will do.

File: src/pages/help-centre/_help-articles.js (hunk 2 above)
Comment (sara-fs) [reply src/pages/help-centre/_help-articles.js]: Sure. 
You are right. No need to write an extra component for it. Thank you.

**Pull request merged**